app/src/common/shared/com/igalia/wolvic/ui/OffscreenDisplay.java

```diff
@@ -85,6 +85,9 @@ public void onResume() {
             Display defaultDisplay = manager.getDisplay(Display.DEFAULT_DISPLAY);
 
             int flags = DisplayManager.VIRTUAL_DISPLAY_FLAG_OWN_CONTENT_ONLY;
+            // TODO: getRealMetrics(DisplayMetrics) in Display has been deprecated.
+            //  Use WindowMetrics#getBounds() to get the dimensions of the application window.
+            //  Use WindowMetrics#getDensity() to get the density of the application window.
```

Comment:
Mind opening an issue for this and link it here?

Reply:
https://github.com/Igalia/wolvic/issues/799